protos/feast/core/CoreService.proto

```diff
@@ -71,6 +75,22 @@ service CoreService {
     // - Changes to feature name and type
     rpc ApplyFeatureSet (ApplyFeatureSetRequest) returns (ApplyFeatureSetResponse);
 
+    // Create or update and existing entity.
+    //
+    // This function is idempotent - it will not create a new entity if schema does not change.
+    // Schema changes will update the entity if the changes are valid.
+    // Following changes are not valid:
+    // - Changes to name
+    // - Changes to type
+    // - Changes to description
```

Comment:
We cant change the description? is this a mistake?

Reply:
I was thinking in the perspective of sharing the use of an entity. If the description is changed after another user has started using it, it may confused him/her about the entity being used. We could allow changing the description, but we'll need to find a way to ensure the description is an appropriate one each time it's changed and also made known to users.

Reply:
How do we ensure the first description is appropriate? Are we supposed to police that? I dont think this is Feast's problem.

Reply:
Yeah, would be difficult. I'll amend this.